validation/validationSchema.tsx

```diff
@@ -0,0 +1,66 @@
+import * as yup from "yup";
+import {
+  OwnedOptions,
+  StatusOptions,
+  TypesOfGamesOptions,
+  PlatformOptions,
+} from "@/components/types";
+
+export const validationSchema = yup.object().shape({
+  game_name: yup
+    .string()
+    .required("Name Validation Field is Required")
+    .min(5, "Must be more that 5 letters")
+    .max(255, "tile is too long"),
+  type_of_game: yup
+    .string()
+    .oneOf(
+      [
+        TypesOfGamesOptions.RPG,
+        TypesOfGamesOptions.PLATFORM,
+        TypesOfGamesOptions.MMORPG,
+        TypesOfGamesOptions.SHOOTER,
+        TypesOfGamesOptions.SURVIVAL,
+        TypesOfGamesOptions.BATTLE_ROYAL,
+        TypesOfGamesOptions.RACING,
+        TypesOfGamesOptions.SPORTS,
+        TypesOfGamesOptions.SIMULATION,
+        TypesOfGamesOptions.MMO,
+        TypesOfGamesOptions.HORROR,
+        TypesOfGamesOptions.POINTS_AND_CLICK,
+        TypesOfGamesOptions.COOPERATIVE,
+      ],
+      "Please pick one option"
```

Comment:
this message will be shown in case of someone entering value that does not belong to the provided list of values, so in that case telling `Please pick one option` does not seem to be a correct message. Maybe `Please chose one of the given options` or something similar?